### DISCUSSION TITLE
Cooldown de 1h pour la libération de familier

### DISCUSSION THREAD

**Discussion by BananePlantain** [Crownicles/Crownicles#2375] [Nouvelles idées]
ca fait lgtps que je voulais la poster puis flemme :smile: c'est supprimer le temps d'attente de 1h pour libérer des 1*

**Comment by Ravenclaw-l** [comment]
Il faudrait que le chef de guilde soit le seul à ne pas avoir ce cooldown. 
Ça permettrait d'éviter les mauvaises surprises.

**Comment by OrionYp** [comment]
Je propose de retirer le cooldown de l'aîné aussi

**Comment by BananePlantain** [comment]
+pouvoir libérer en alté

**Reply by Procion06** [reply]
oui bonne idée
on peut envoyer son pet en expé sous alté mais pas le libérer, spa super logique xD
